Package.swift

```diff
@@ -7,9 +7,9 @@ let package = Package(
         .Package(url: "https://github.com/vapor-community/postgresql.git", Version(2,0,0, prereleaseIdentifiers: ["beta"])),
 
         // Swift models, relationships, and querying for NoSQL and SQL databases.
-        .Package(url: "https://github.com/vapor/fluent.git", Version(2,0,0, prereleaseIdentifiers: ["beta"])),
+        .Package(url: "https://github.com/vapor/fluent.git", Version(2,0,0)),
```

Comment:
For a library you probably want to to just set the major version rather than pinning it at 2.0.0